airbyte-integrations/connectors/source-nasa-apod/source_nasa_apod/spec.yaml

```diff
@@ -0,0 +1,50 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/nasa-apod
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: NASA APOD spec
+  type: object
+  required:
+    - api_key
+  properties:
+    api_key:
+      type: string
+      description: API access key used to retrieve data from the NASA APOD API.
+      airbyte_secret: true
+    concept_tags:
+      type: boolean
+      default: false
+      description: >-
+        Indicates whether concept tags should be returned with the rest of the response. 
+        The concept tags are not necessarily included in the explanation, but rather derived
+        from common search tags that are associated with the description text. (Better than
+        just pure text search.) Defaults to False.
+    count:
+      type: integer
+      minimum: 1
+      maximum: 100
+      description: >- 
+        A positive integer, no greater than 100. If this is specified then `count` randomly 
+        chosen images will be returned in a JSON array. Cannot be used in conjunction with 
+        `date` or `start_date` and `end_date`.
+    start_date:
+      type: string
+      description: >-
+        Indicates the start of a date range. All images in the range from `start_date` to 
+        `end_date` will be returned in a JSON array. Cannot be used with `date`.
```

Comment:
Update this description. If you keep this empty it will return all images from the beginning?

Reply:
Okey, noted! If we keep `start_date` empty then we cannot specify `end_date`. By default, if we call the api with no parameters it default to retrieve only the image corresponding to today.

The only way to get all images from the beginning is to set `start_date` and `end_date` accordingly.

Reply:
In the case that one does not specify `end_date`, then the API default to today, thus, by only specifying `start_date` to the very first possible date (1995-06-16), we would retrieve all available images.